### PR TITLE
add new snippets for October per bug 1408280

### PR DESCRIPTION
**Pull request by peiying2**
Ignore the branch name, I thought it was for November, :).  

https://bugzilla.mozilla.org/show_bug.cgi?id=1408280

### CODE REVIEW

**Comment by flodolo** [comment]
> Ignore the branch name, I thought it was for November, :).

Just remember that for that next time ;-)

File: en-US/snippets/2017/oct2017.lang [hunk 1]
```diff
@@ -0,0 +1,99 @@
+## NOTE: Reference bug 1408280. Deadline: 25 October, 2017.
+## NOTE: Guidelines on localizing snippets: https://goo.gl/nEhdt4. The tone is QUAC (Quick, Untamed, Alert, Clever). Do your best to make it local and culturally relevant.
+## NOTE: No character length limit, but try not to exceed English length.
+
+
+# Snippet for Firefox quantum
+# DeLorean refers to the time machine from the Back to the Future films. If there are better known time machine reference in your culture, feel free to swap it out.
+# Link: https://www.mozilla.org/firefox/quantum/
```
Comment (flodolo) [inline en-US/snippets/2017/oct2017.lang]: Either this link is wrong, or they are pointint 7 out of 8 languages to a page available only in English. The new Quantum landing page will be at /firefox, and available only on Nov 14.

File: en-US/snippets/2017/oct2017.lang [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+;See the future of Firefox, no DeLorean required. <a>Check out Quantum.</a>
+See the future of Firefox, no DeLorean required. <a>Check out Quantum.</a>
+
+
+# Snippet for Firefox quantum
+# “aka” is the acronym for “also known as”. Feel free to substitute it with the equivalent in your language.
+# Link: https://www.mozilla.org/firefox/quantum/
```
Comment (flodolo) [inline en-US/snippets/2017/oct2017.lang]: Same note

File: en-US/snippets/2017/oct2017.lang [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+;Don’t miss out — Firefox Quantum (aka the best Firefox ever) launches soon. <a>Get notified when it becomes available.</a>
+Don’t miss out — Firefox Quantum (aka the best Firefox ever) launches soon. <a>Get notified when it becomes available.</a>
+
+
+# Snippet for Firefox features
+# The repeat use of the word "fast" is to convey to users that Firefox is the speedest it's ever been.
+;The new Firefox can be summed up in two words: faster and faster. OK, maybe that’s the same word twice.<a>Learn More.</a>
```
Comment (flodolo) [inline en-US/snippets/2017/oct2017.lang]: Missing a space after 'twice.'

File: en-US/snippets/2017/oct2017.lang [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+# Snippet for Firefox features
+# Tab hoarding is the practice of having a lot of browser tabs open.
+;<a>Firefox is refreshingly fast,</a> with quicker startup times, fewer crashes and better at making tab hoarders happy (you know who you are.)
+<a>Firefox is refreshingly fast,</a> with quicker startup times, fewer crashes and better at making tab hoarders happy (you know who you are.)
+
+
+# Snippet for Firefox
+;Firefox is the only browser that’s made by a non-profit, so it doesn’t sell access to your every click. #Nowyouknow
+Firefox is the only browser that’s made by a non-profit, so it doesn’t sell access to your every click. #Nowyouknow
```
Comment (flodolo) [inline en-US/snippets/2017/oct2017.lang]: Are they supposed to localize the hashtag?

File: en-US/snippets/2017/oct2017.lang [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+
+# Snippet for Firefox
+;Mozilla is a champion of the open web. Firefox is made by Mozilla. So every time you use Firefox, (like right now) you’re helping to make the web better for everyone.
+Mozilla is a champion of the open web. Firefox is made by Mozilla. So every time you use Firefox, (like right now) you’re helping to make the web better for everyone.
+
+
+# Snippet for form fill feature
+# Link: https://support.mozilla.org/kb/control-whether-firefox-automatically-fills-forms
+;Firefox can <a>fill out forms for you,</a> so you have more time to do important stuff online, like look at memes.
+Firefox can <a>fill out forms for you,</a> so you have more time to do important stuff online, like look at memes.
```
Comment (flodolo) [inline en-US/snippets/2017/oct2017.lang]: No sense in promoting a feature, with a localized snippet, that is only available to en-US builds in US territory.

File: en-US/snippets/2017/oct2017.lang [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+
+
+# Snippet for add-ons
+# Link: https://addons.mozilla.org/firefox/collections/mozilla/essential-extensions/
+;Powerful privacy tools, media enhancers and more, our Essential extensions collection is loaded with awesome add-ons. <a>Check them out.</a>
+Powerful privacy tools, media enhancers and more, our Essential extensions collection is loaded with awesome add-ons. <a>Check them out.</a>
+
+
+# Snippet for add-ons
+# Fidget spinners are the lastest craze in the US. Feel free to swap out with something that's very popular locally at the moment.
```
Comment (flodolo) [inline en-US/snippets/2017/oct2017.lang]: Not just US :-\

File: en-US/snippets/2017/oct2017.lang [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+# Link: https://addons.mozilla.org/firefox/collections/mozilla/trending-add-ons/
+;Don't let your friends see you with an outdated add-on. <a>Explore our trending add-ons collection</a> for the latest in excellent Firefox extensions.
+Don't let your friends see you with an outdated add-on. <a>Explore our trending add-ons collection</a> for the latest in excellent Firefox extensions.
+
+
+# Snippet for add-ons
+# "Take one for a spin" means to try something out.
+# Link: https://addons.mozilla.org/firefox/collections/mozilla/trending-add-ons/
+;Our trending extensions collection is filled with emerging add-ons that are gaining traction among Firefox users. <a> Take one for a spin.</a>
+Our trending extensions collection is filled with emerging add-ons that are gaining traction among Firefox users. <a> Take one for a spin.</a>
```
Comment (flodolo) [inline en-US/snippets/2017/oct2017.lang]: Remove extra space after `<a>`

File: en-US/snippets/2017/oct2017.lang [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+
+
+# Snippet on Send Tabs
+# Roam here means mobility between devices.
```
Comment (flodolo) [inline en-US/snippets/2017/oct2017.lang]: Not sure this comment is helpful, it suggests to lose the "tie down vs roam free" part.

File: en-US/snippets/2017/oct2017.lang [hunk 9]
```diff
@@ -0,0 +1,99 @@
...
+# “Free your mind” is a way of saying let go of your worries.
+# Link: https://www.mozilla.org/firefox/features/send-tabs/
+;Send Tabs makes device-to-device sharing simple. Free your tabs and your mind will follow. <a>Learn more.</a>
+Send Tabs makes device-to-device sharing simple. Free your tabs and your mind will follow. <a>Learn more.</a>
+
+
+# Snippet on Send Tabs
+# “Free your mind” is a way of saying let go of your worries.
+# Link: https://www.mozilla.org/firefox/features/send-tabs/
+;Find an article on your desktop that would go nicely with your morning commute? Don't bother emailing it to yourself, <a>use Send Tabs instead. </a>
```
Comment (flodolo) [inline en-US/snippets/2017/oct2017.lang]: Remove extra space before `</a>`.

File: en-US/snippets/2017/oct2017.lang (hunk 1 above)
Comment (peiying2) [reply en-US/snippets/2017/oct2017.lang]: Confirmed to change it to landing page instead https://www.mozilla.org/firefox/

File: en-US/snippets/2017/oct2017.lang (hunk 2 above)
Comment (peiying2) [reply en-US/snippets/2017/oct2017.lang]: same here too.

File: en-US/snippets/2017/oct2017.lang (hunk 5 above)
Comment (peiying2) [reply en-US/snippets/2017/oct2017.lang]: i see German with a screenshot, but not sure about other languages on this one. the article is localized in other languages.

File: en-US/snippets/2017/oct2017.lang (hunk 4 above)
Comment (peiying2) [reply en-US/snippets/2017/oct2017.lang]: Probably not. We have had this type before and it was left in English. let me make a note of it

File: en-US/snippets/2017/oct2017.lang (hunk 5 above)
Comment (flodolo) [reply en-US/snippets/2017/oct2017.lang]: It's not the article. The feature is only enabled for en-US in US.

**Pull request merged**